generator/template_repo.go

```diff
@@ -153,7 +155,9 @@ func (t *Repository) LoadDir(templatePath string) error {
 		if strings.HasSuffix(path, ".gotmpl") {
 			assetName := strings.TrimPrefix(path, templatePath)
 			if data, err := ioutil.ReadFile(path); err == nil {
-				t.AddFile(assetName, string(data))
+				if err := t.AddFile(assetName, string(data)); err != nil {
```

Comment:
I would just stop process everything, even fatal log message would be enough, since the overall situation is just not normal. I had a hard time trying to figure why my server template has disappeared ;)